test/ui-xcm/ui-main.BNC.deposit.test.ts

```diff
@@ -115,6 +115,7 @@ describe("UI XCM tests - BNC", () => {
 
     //TODO: Remove when clarified how to setup tokens on Bifrost.
     const mgaSdk = MangataSDK.instance([mangata.uri]);
+    // @ts-ignore
```

Comment:
@MariuszSzpyt 
thinking of removing those xcm tests, since no xcm will be possible on Rolldown.
For now - ts ignore to avoid ts compiler errors.

Reply:
Sure! soon we will split FE repo into rollup and kusama app so will review tests, however ui-xcm is not used anymore - xcm tests for current app are in test/microapps-ui-xcm. I will clean it up once we will make rollup/kusama app separation!